### PR TITLE
Plain JavaScript

### PR DESCRIPTION
**Pull request by imubarak234**
My partner and I implemented the Awesomes book application, we wrote the add and remove buttons with appropriate functionalities. We also implemented the save to local storage options.

### CODE REVIEW

File: books.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+      const auth = `#count${count1} h4`;
+      const nodes = document.getElementById(ids);
+      const names = document.querySelector(nam);
+      const author = document.querySelector(auth);
+      removeList(names.innerHTML, author.innerHTML);
+      nodes.parentNode.removeChild(nodes);
+    }
+    count1 += 1;
+  });
+}
```
Comment (mikethreels) [inline books.js]: - [ ] good job setting up the project 👍 however after populating and refreshing the list the user is not able to remove the existing books anymore. when adding a `console.log()` in the `addEventListener` for removing the item it looks like the event is not called on click.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: books.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+const next = document.getElementById('addButton');
```
Comment (carlos-ssh) [inline books.js:72]: - Update your ``README`` file.

File: books.js [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+function removeList(name, author) {
+  for (let x = 0; x < bookList.length; x += 1) {
+    if ((bookList[x].title === name) && (bookList[x].author === author)) {
+      bookList.splice(x, 1);
+    }
+  }
+}
+
+function removeBook(args) {
```
Comment (carlos-ssh) [inline books.js:52]: **Kindly fix this issue displayed in console** 
- When i tried to remove an item and this error displayed:
Screen Shot 2021-11-23 at 7 34 44

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
# 🦺 🚧 🚥 CHANGES REQUIRED 🚥 🚧 🦺
#

Hi @imubarak234 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._ 


changesrequested
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Laxmi and Mubarak, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**